### DISCUSSION TITLE
llama.cpp Android problem

### DISCUSSION THREAD

**Discussion by ChanwooCho** [ggml-org/llama.cpp#8163] [Q&A]
I followed the instructions in the readme to set up Android, 
but I encountered an error saying: "CANNOT LINK EXECUTABLE './llama-cli': library 'libomp.so' not found: needed by main executable."

It was working fine with the previous version.

**Comment by micsthepick** [comment]
can you provide more details about your environment? Have you updated packages recently?

**Comment by FranzKafkaYu** [comment]
@ChanwooCho you need package libomp.so into you apk,you can get it in your NDK.

**Reply by theoctopusride** [reply]
Can you explain how you did this?

**Reply by FranzKafkaYu** [reply]
> Can you explain how you did this?

simply you can find the library in your NDK，plese choose the right version.And import this library to device.
